aptos-move/aptos-aggregator/src/delta_change_set.rs

```diff
@@ -338,13 +343,35 @@ impl DeltaChangeSet {
         self,
         state_view: &impl StateView,
     ) -> anyhow::Result<WriteSet, VMStatus> {
-        let materialized_write_set = self
-            .take(state_view)
-            .expect("something terrible happened when applying aggregator deltas");
+        match self.take(state_view) {
```

Comment:
Hey @georgemitenkov, it was actually me who did this but now it confused me - can we add a comment to take (that it materializes) or change the name to take_materialized.